.github/workflows/auto-bump-datadog-agent.yml

```diff
@@ -0,0 +1,68 @@
+name: Open datadog-agent PR
+run-name:
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  open_bump_pr:
+    if: github.event.pull_request.merged == true && !contains(github.event.pull_request.labels.*.name, 'no-auto-bump')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Token
+        uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ vars.DATADOG_APP_ID }}
+          private-key: ${{ secrets.DATADOG_APP_PRIVATE_KEY }}
+          repositories: datadog-agent
+
+      - name: Clone datadog-agent repo
+        uses: actions/checkout@v4
+        with:
+          repository: datadog/datadog-agent
+          persist-credentials: false
+          path: datadog-agent
+          token: ${{ steps.app-token.outputs.token }}
+          fetch-depth: 0
+      - name: Get previous SHA
+        id: get-previous-sha
+        working-directory: datadog-agent
+        run: |
+          PREVIOUS_SHA=$(cat .gitlab-ci.yml | grep TEST_INFRA_DEFINITIONS_BUILDIMAGES: | awk -F " " '{print $NF}')
+          echo PREVIOUS_SHA=${PREVIOUS_SHA} >> "${GITHUB_OUTPUT}"
+      - name: Setup Python3
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11.5"
+          cache: "pip"
+
+      - name: Install python deps
+        working-directory: datadog-agent
+        run: |
+          pip3 install -r requirements.txt
+
+      - name: Update test-infra-defintions version
+        working-directory: datadog-agent
+        run: |
+          inv -e buildimages.update-test-infra-definitions --commit-sha ${{ github.event.pull_request.head.sha }}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Does this require a `datadog-agent` version?

Reply:
It requires the PR from @julien-lebot : https://github.com/DataDog/datadog-agent/pull/23918
But this is not a problem since it always creates PR targetting `main` of datadog-agent